.github/workflows/release.yml

```diff
@@ -19,8 +19,18 @@ jobs:
 
       - name: Setup
         run: npm install -g semantic-release @semantic-release/github @semantic-release/changelog @semantic-release/commit-analyzer @semantic-release/npm @semantic-release/git @semantic-release/release-notes-generator
-  
+      
+      - name: Set up python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      
+      - name: Install setuptools
+        run: python -m pip install --upgrade setuptools wheel twine
+      
       - name: Rlease
-        run: npx semantic-release  
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+          GH_TOKEN: ${{ secrets.GH_TOKEN }}
+          PYPI_TOKEN: ${{ secrets.PYPI_TOKEN }}
+          TWINE_REPOSITORY: pycasbin
```

Comment:
TWINE_REPOSITORY is not required. This environment variable can be used, to publish a package to a different python package index than pypi

Reply:
Thank you!I fixed it. :)